docs/user/alerts.md

```diff
@@ -0,0 +1,75 @@
+# Cluster Autoscaler Alerts
+
+This document describes the alerts generated by the Cluster Autoscaler Operator,
+their possible causes, and suggested resolutions.
+
+
+## ClusterAutoscalerUnschedulablePods
+The cluster autoscaler is unable to scale up and is alerting that there are
+unschedulable pods because of this condition.
+
+### Query
+```
+# for: 20m
+cluster_autoscaler_unschedulable_pods_count{service="cluster-autoscaler-default"} > 0
+```
+
+### Possible Causes
+* The cluster autoscaler does not have enough resources to increase the size of
```

Comment:
How about

"The autoscaler is unable to create new machines due to resource limits on MachineAutoscalers"

Reply:
this makes me think we need 2 entries here, the one you are proposing and one about the ClusterAutoscaler resources. i think you can hit this alert if your MachineAutoscaler doesn't have enough replicas or if your ClusterAutoscaler doesn't define enough nodes/cpu/ram. wdyt?

Reply:
I concur.